webapp/masterclasses.py

```diff
@@ -1,6 +1,16 @@
+import os
 from datetime import datetime
 import flask
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+
 from webapp.spreadsheet import get_sheet, MissingCredential
+from webapp.models import PreviousSession, UpcomingSession
+
+db_engine = create_engine(os.getenv("DATABASE_URL"))
```

Comment:
just a side note, `create_engine` can take in an `URL` object from `sqlalchemy` which allows you to programatically set connection fields like below instead of passing in a url string. I think in the context of this PR it doesn't matter too much which one to use, but the `URL` object approach does allow easier management of connection details with more complex environment setups.

```
URL.create(
    "postgressql+psycopg",
    username=DB_USER,
    password=DB_PASS,
    host=DB_HOST,
    database=DB_NAME,
    port=DB_PORT
)
```